packages/gatsby-recipes/src/providers/gatsby/utils/build-plugin-node.js

```diff
@@ -6,13 +6,16 @@ module.exports = ({ name, options, key }) => {
     return t.stringLiteral(name)
   }
 
-  const pluginWithOptions = template(`
+  const pluginWithOptions = template(
+    `
     const foo = {
       resolve: '${name}',
       options: ${JSON.stringify(options, null, 2)},
       ${key ? `__key: "` + key + `"` : ``}
     }
-  `)()
+  `,
+    { placeholderPattern: false }
```

Comment:
does this look right @johno? It was erroring when I passed in integers as option values otherwise.